src/installer/corehost/cli/apphost/apphost.windows.cpp

```diff
@@ -61,10 +61,11 @@ namespace
         if (pal::getenv(_X("DOTNET_DISABLE_GUI_ERRORS"), &gui_errors_disabled) && pal::xtoi(gui_errors_disabled.c_str()) == 1)
             return;
 
-        pal::string_t dialogMsg = _X("To run this application, you must install .NET Core.\n\n");
+        pal::string_t dialogMsg;
```

Comment:
Doesn't this mean that the missing framework case where 3.0+ hostfxr is present will no longer have the `To run this application...` portion of the message?

Reply:
Right, I'm adding it back. Although, I think that then the custom error message will look too repetitive `dialogMsg = "To run this application, you must install .NET Core.\n\nTo run this application, you need to install a newer version of .NET Core."`.

Reply:
Yeah, I'd think we shouldn't have it for the custom message case (i.e. the custom message replaces `To run this application, you must install .NET Core.\n\n` instead of adding to it).